packages/expo-av/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 
 ### 🐛 Bug fixes
 
+- Do not publish native tests to npm. ([#8881](https://github.com/expo/expo/pull/8881) by [@mczernek](https://github.com/mczernek))
+
```

Comment:
It sounds more like a warrant than changelog entry 😄
Also, since this is not user-facing change and more related with expo-module-scripts, I wouldn't add it here but in expo-module-scripts instead.
```suggestion
```